R/get_ssm_by_regions.R

```diff
@@ -84,22 +96,16 @@ get_ssm_by_regions = function(regions_list,
     rn = regions_bed[["name"]]
   }
 
-  if(basic_columns){
-    #this must always force the output to be the standard set.
-    #hence, return everything after binding into one data frame
-    print("bind_rows")
-    return(bind_rows(region_mafs))
-  }
   tibbled_data = tibble(region_mafs, region_name = rn)
   unnested_df = tibbled_data %>%
     unnest_longer(region_mafs)
+  
   if(streamlined){
     unlisted_df = mutate(unnested_df, start = region_mafs$Start_Position, sample_id = region_mafs$Tumor_Sample_Barcode) %>%
       dplyr::select(start, sample_id, region_name)
-
-  }else{
-    unlisted_df = mutate(unnested_df, Chromosome = region_mafs$Chromosome, End_Position = region_mafs$End_Position, Start_Position = region_mafs$Start_Position, Tumor_Sample_Barcode = region_mafs$Tumor_Sample_Barcode) %>%
-      dplyr::select(Chromosome, Start_Position, End_Position, Tumor_Sample_Barcode, region_name)
-  }
+    }else{
```

Comment:
Do you happen to have an example how this used to work and how this works now? I think the `region_mafs` is a list of dfs and I think it might need do.call to bind everything properly - but maybe I am not correct 🤦